src/content/about/blog/oscal-first-deep-cleaning.md

```diff
@@ -0,0 +1,24 @@
+---
+title: OSCAL's Fall Cleaning
+---
+
+Welcome to the Open Security Controls Assessment Language (OSCAL) Blog, open to the NIST OSCAL Team and to the community! 
```

Comment:
"Team" should not be 'team'

Reply:
NIST OSCAL Team is a formal noun, where as team is not. Do you really want it to be team?